### PR TITLE
Add datadog eventing

### PR DESCRIPTION
**Pull request by zmmille2**
Add dependency injected datadog service and supporting classes.

### CODE REVIEW

File: HelloApi/HelloApi/Filters/EventFilter.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace HelloApi.Filters
+{
+    public class EventFilter : ActionFilterAttribute, IActionFilter
+    {
+        public EventFilter() { }
+
+        void IActionFilter.OnActionExecuting(ActionExecutingContext filterContext)
+        {
+            using (var eventLogger = (IEventLogger) filterContext.HttpContext.GetService(typeof(IEventLogger))) {
```
Comment (tbarlow12) [inline HelloApi/HelloApi/Filters/EventFilter.cs:14]: nit: `{` on separate line

**Review by tbarlow12 — APPROVED** [review]
LGTM

File: HelloApi/HelloApi/HelloApi.csproj [hunk 2]
```diff
@@ -273,6 +281,11 @@
         </WebProjectProperties>
       </FlavorProperties>
     </VisualStudio>
+    <MonoDevelop>
+      <Properties>
+        <XspParameters Port="8080" Address="127.0.0.1" SslMode="None" SslProtocol="Default" KeyType="None" CertFile="" KeyFile="" PasswordOptions="None" Password="" Verbose="True" />
+      </Properties>
+    </MonoDevelop>
```
Comment (wbreza) [inline HelloApi/HelloApi/HelloApi.csproj:288]: Any idea what this is for?

**Review by wbreza — APPROVED** [review]
Looking good!

File: HelloApi/HelloApi/HelloApi.csproj (hunk 2 above)
Comment (zmmille2) [reply HelloApi/HelloApi/HelloApi.csproj:309]: https://stackoverflow.com/questions/50294691/what-is-the-xspparameters-element-that-visual-studio-for-mac-adds-to-a-csproj-f

Seems like we can remove.

**Comment by zmmille2** [comment]
Closes #17.

**Pull request merged**